ConsoleApp/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace ConsoleApp
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int a = 1, 
+                b = 2;
```

Comment:
you should make these two variables, been provided by the user #2

Reply:
Ok